### PR TITLE
Adds favicon

### LINKED ISSUE #207: Set Politeia's logo as favicon
The favicon and Polieia's logo is different. In my opinion, they should be the same, for consistency.

screen shot 2018-02-07 at 21 10 15

### PR DESCRIPTION
**Pull request by rrecuero**
Closes #207

### CODE REVIEW

**Comment by tiagoalvesdulce** [comment]
Hey @rrecuero, thanks for your PR, I liked the new favicon. Can you just go to: https://github.com/decred/politeiagui/blob/9c92a0fda6a4e64c4c095c855f198d285c22e925/public/index.html#L12 and change it to: `<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=2">` before we merge it? This `?v=2` makes sure our users will download the right `favicon` without having to clear the cache. Also, edit the comment as I said on your previous PR.

**Comment by rrecuero** [comment]
@tiagoalvesdulce thanks forgot the cachestamping

File: public/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ic?v=2o">
```
Comment (tiagoalvesdulce) [inline public/index.html]: You got a typo here @rrecuero, should be `...favicon.ico?v=2`

**Comment by rrecuero** [comment]
@tiagoalvesdulce sorry for the typo. Fixed and updated the comment

**Pull request merged**